src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else if (days <= 3) {
+    return days * 40;
```

Comment:
Do you need else...if?
Your code won't go further after return anyway.
Also, let's create descriptive variables for all the main values to make your code cleaner and more understandable